WordPress/src/main/res/layout/reader_cardview_post_new.xml

```diff
@@ -187,8 +192,8 @@
             style="@style/ReaderTextView.Post.New.FooterActionItem"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
-            android:layout_marginTop="@dimen/margin_medium"
-            android:layout_marginEnd="@dimen/margin_extra_large"
+            android:paddingTop="@dimen/margin_medium"
+            android:paddingBottom="@dimen/margin_medium"
```

Comment:
💡  Since these paddings are now part of the "component" itself and exactly the same for all Footer Action items, would it make sense for them to be defined inside the `ReaderTextView.Post.New.FooterActionItem` style instead of each individual button?

Longer explanation:
The way I think of it is: if something is external to the component (like a margin, which is part of the specific screen layout, position, or even width/height) I prefer defining it for each instance because it's the screen layout "job" to set that. On the other hand, if something is more internal (padding, text style, background color, etc) to the component, it makes sense to be part of the component definition itself, which for Android Views is the `style`.